ctapipe/image/leakage.py

```diff
@@ -43,8 +43,8 @@ def leakage(geom, image, cleaning_mask):
     size = np.sum(image[cleaning_mask])
 
     return LeakageContainer(
-        leakage1_pixel=leakage_pixel1 / geom.n_pixels,
-        leakage2_pixel=leakage_pixel2 / geom.n_pixels,
-        leakage1_intensity=leakage_intensity1 / size,
-        leakage2_intensity=leakage_intensity2 / size,
+        fraction_1pix=leakage_pixel1 / geom.n_pixels,
```

Comment:
both are fractions, one is the fraction of pixels, the other the fraction of p.e. So maybe::
```
pixel_fraction_1
pixel_fraction_2
pe_fraction_1
pe_fraction_2
```
?

Reply:
how about this?: 

- `leakage_fraction_intensity_1pix`
- `leakage_fraction_intensity_2pix`
- `leakage_fraction_pixels_1pix`
- `leakage_fraction_pixels_2pix`

Or we just define "leakage" as a fraction, and we can remove `_fraction` from all of them? (which brings us back to the first updated version, I think!)

Reply:
Maybe width instead of pix to make it clearer? Naming things....

leakage_pixels_width_1
leakage_pixels_width_2
leakage_intensity_width_1
leakage_intensity_width_2

I don't like the `1pix` `2pix` part.

Reply:
yeah, I had also considered "neighbors", but that was a bit long... Width is fine.

Reply:
though it's not clear the unit of width (not hillas width, for example)

Reply:
I updated the docstring (which was basically useless before) to explain it better.